modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -39,6 +40,29 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.12.1
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Prevent 507 Insufficient Storage on 32-Bit Systems
+
+With the introduction of https://github.com/owncloud/core/pull/40567 compatibility with 32-bit systems broke as we are now casting $freeSpace to int and this caused an integer overflow on such systems when the free space was above the max supported value. We added therefore an additional check for 32-bit systems in QuotaPlugin.php. https://github.com/owncloud/core/pull/40709[#40709]
+
+=== User-Key Encryption Is No Longer Auto-Enabled
+
+Executing the "encryption:encrypt-all" occ command will no longer auto-enable user-key encryption. https://github.com/owncloud/core/pull/40702[#40702]
+
+=== Package firebase/php-jwt in files_texteditor
+
+Since version 2.5.0 of the files_texteditor app we use Firebase\JWT\JWT, but this dependency was not directly shipped with files_texteditor. This was not an issue in case the files_external and WOPI apps were enabled as they include the same dependency. However, in case both apps were disabled the functionality of the files_texteditor app broke. We now ship this dependency in files_texteditor as well. https://github.com/owncloud/files_texteditor/pull/389[files_texteditor#389]
```

Comment:
```suggestion
Since version 2.5.0 of the files_texteditor app we use Firebase\JWT\JWT, but this dependency was not directly shipped with files_texteditor. This was not an issue if the files_external or WOPI app(s) were enabled as they include the same dependency. However, if both apps were disabled the functionality of the files_texteditor app broke. We now ship this dependency in files_texteditor as well. https://github.com/owncloud/files_texteditor/pull/389[files_texteditor#389]
```